_activities/gsoc.md

```diff
@@ -49,100 +49,100 @@ For further information, feel free to contact the HSF GSoC admins or join our op
 
 ### Student pages
 
-The CERN-HSF 2022 student blogs are now available. [Check them out!]({{ site.baseurl }}/gsoc/2022/blogs.html)
+The CERN-HSF 2023 student blogs are now available. [Check them out!]({{ site.baseurl }}/gsoc/2023/blogs.html)
 
-Student blogs are mandatory for receiving a positive evaluation in CERN-HSF starting with 2022 GSoC edition! If you are a CERN-HSF student, please read this [blog]({{ site.baseurl }}/gsoc/blogs/2022/blog_HSF_TheAdmins.html).
+Student blogs are mandatory for receiving a positive evaluation in CERN-HSF! If you are a CERN-HSF student, please read this [blog]({{ site.baseurl }}/gsoc/blogs/2022/blog_HSF_TheAdmins.html).
```

Comment:
Apparently this is not true!

Reply:
indeed. we haven't been pushy enough last time